package.json

```diff
@@ -246,6 +246,7 @@
     "db:seed": "cross-env NODE_PATH=./ node api/migrations/seed/index.js",
     "db:reset": "yarn run db:drop && yarn run db:migrate && yarn run db:seed",
     "rethinkdb:migrate": "rethinkdb-migrate -f ./api/migrations/config.js",
+    "rethinkdb:migrate:generate": "rethinkdb-migrate create -f ./api/migrations/config.js --",
```

Comment:
Nitpick: Let's call this `rethinkdb:migrate:create` in order to be consistent with the actual command run

Reply:
Good call.